cmd/grype/cli/commands/db_check.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/anchore/clio"
-	legacyDistribution "github.com/anchore/grype/grype/db/legacy/distribution"
+	legacyDistribution "github.com/anchore/grype/grype/db/v5/distribution"
```

Comment:
the smallest of nits: it would be nice if this was `v5distribution` or something with `v5` rather than legacy (it was really helpful for me when going through the code to see "v5" where it was used and not just the import)

Reply:
I didn't catch this before automerge kicked in -- I'll incorporate on a follow up PR